docs/errors/usage_errors.md

```diff
@@ -882,7 +882,7 @@ except PydanticUserError as exc_info:
     assert exc_info.code == 'multiple-field-serializers'
 ```
 
-## Invalid annotated type {#invalid_annotated_type}
```

Comment:
Also noticed this, maybe it's a bad idea to change this now (or maybe we need to add it to a redirect in the cloudflare worker) but this was the only one using snake- rather than kebab-case.